source/_integrations/mazda.markdown

```diff
@@ -26,13 +27,34 @@ This integration requires an active Mazda Connected Services subscription and a
 - CX-9: 2021+
 - CX-5: 2021+
 
-This integration provides the following platforms:
+## Platforms
```

Comment:
Please move extensive platform descriptions below the integration installation / set up instructions.

The introduction should only provide a brief overview.

Reply:
Fixed.